crates/networking/discv5/Cargo.toml

```
@@ -10,9 +10,18 @@ rust-version.workspace = true
 version.workspace = true
 
 [dependencies]
+alloy-primitives.workspace = true
 anyhow.workspace = true
 discv5.workspace = true
 futures.workspace = true
 libp2p.workspace = true
 tokio.workspace = true
 tracing.workspace = true
+
+# external crates 
+ethereum_ssz = "0.9.0"
+ethereum_ssz_derive = "1.0.0-beta.2"
```

```suggestion
ethereum_ssz.workspace = true
ethereum_ssz_derive.workspace = true
```

We should be importing from the workspace, and this should be sorted in the list above